ev3dev2/motor.py

```diff
@@ -1774,6 +1791,48 @@ def _block(self):
         self.wait_until_not_moving()
 
 
+# line follower classes
+class LineFollowError(Exception):
```

Comment:
`LineFollowErrorLostLine`? It's weird that this name is a substring of the name of another error but neither extends the other.

Reply:
`LineFollowErrorLostLine` sounds good to me.  We could also have both of these inherit from a base `LineFollowError` class.